ConsoleWriter.cs

```diff
@@ -10,13 +11,33 @@ public void WriteLine(string s)
             Console.WriteLine(s);
         }
         
-        public void PrintBoard(char[] boardState)
+        public void PrintBoard(List<object> boardList)
         {
-            Console.WriteLine(" {0} | {1} | {2} ", boardState[0], boardState[1], boardState[2]);
+            Console.WriteLine(" {0} | {1} | {2} ", boardList[0], boardList[1], boardList[2]);
```

Comment:
Is there a way here that the writer can build the board based on the size of the array?